packages/gatsby-cli/src/create-cli.js

```diff
@@ -346,6 +346,21 @@ module.exports = argv => {
         }
       ),
     })
+    .command({
+      command: `plugin`,
+      desc: `Welcome to the Gatsby source plugin help center!
+
+      Resources:
+- Walk through the tutorial (https://www.gatsbyjs.org/docs/source-plugin-tutorial/)
+- Read requirements for Gatsby packages (file and naming conventions) (https://www.gatsbyjs.org/docs/how-plugins-work/)
+- Submit to plugin library (https://www.gatsbyjs.org/contributing/submit-to-plugin-library/)
+- Join Discord #need-help channel to ask questions
```

Comment:
There is a new #plugin-authoring channel in Discord we could mention here!

Reply:
Nice it has been created since I made the PR, thanks 😄